third_party/move/move-prover/boogie-backend/src/spec_translator.rs

```diff
@@ -908,7 +908,13 @@ impl<'env> SpecTranslator<'env> {
                     "currently `TRACE(..)` cannot be used in spec functions or in lets",
                 )
             },
-            Operation::MoveFunction(_, _) | Operation::BorrowGlobal(_) | Operation::Old => {
+            Operation::MoveFunction(_, _)
+            | Operation::BorrowGlobal(_)
+            | Operation::MoveTo
+            | Operation::MoveFrom
+            | Operation::Freeze
+            | Operation::Abort
+            | Operation::Old => {
```

Comment:
would this have any impact on the prover?

Reply:
No, since `spec_translator` translates the specification expression language subset, and those new operators belong to the implementation subset. 

One important step of the new compiler is unifying the expression language of the spec and the impl worlds. Some operators are not defined in both worlds.